tests/0290-oasis_FindObjects.phpt

```diff
@@ -138,17 +135,7 @@ int(0)
 int(0)
 int(0)
 int(0)
-int(1)
-dump object %d: array(3) {
-  [16]=>
-  string(8) "PHP Test"
-  [17]=>
-  string(12) "Hello World!"
-  [3]=>
-  string(37) "Test Label - %A"
-}
```

Comment:
I think this test should definitely check for the object's attributes to ensure it's the correct object.

Reply:
The problem is that different HSMs lead to different output. I performed the test 0290 on another HSM that is not SoftHSM2, for this case, I cannot not get this output.

Same for the 0291, SoftHSM2 does not provide the same output.

I agree that it is a "weak" check, but it seems to be "good enough" : it was already helpful to detect some segfault. I prefer a "weak" test that can be improved later on than "no test".

Before having a "hardened" design, I'd like to use these tests in order to perform some more robustness analysis with Valgrind once they'll be set.